ArduPlane/Parameters.cpp

```diff
@@ -1225,6 +1225,37 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(guidedHeading, "GUIDED_", 28, ParametersG2, AC_PID),
 #endif // OFFBOARD_GUIDED == ENABLED
 
+    // @Param: MAN_EXPO_ROLL
+    // @DisplayName: Manual control expo for roll
+    // @Description: Percentage exponential for roll input in MANUAL, ACRO and TRAINING modes
+    // @Range: 0 100
```

Comment:
Not a big issue, but I think these should be in the range 0 to 1 for better alignment with expo params in copter.  Just to keep them consistant between vehicles.

Reply:
And in  transmitters